bitnami/keycloak/templates/configmap-env-vars.yaml

```
@@ -31,7 +31,7 @@ data:
   {{- end }}
   {{- if .Values.ingress.enabled }}
   KC_HOSTNAME_STRICT: {{ ternary "true" "false" .Values.hostnameStrict | quote }}
-{{- if .Values.ingress.hostname }}
+{{- if and .Values.ingress.hostname (eq .Values.hostnameStrict true) }}
```

Adding the "eq true" condition is a bit redundant. Can you review this change? 

```suggestion
{{- if and .Values.ingress.hostname .Values.hostnameStrict }}
```